composer/profiler/profiler.py

```diff
@@ -214,11 +224,11 @@ def get_action(self, batch_idx: int) -> ProfilerAction:
         return ProfilerAction.ACTIVE
 
     @property
-    def event_handlers(self):
+    def handlers(self):
```

Comment:
If the trainer is directly constructing the profiler callbacks, then I think this can be kept as `event_handlers`.